index.js

```diff
@@ -35,6 +35,7 @@ module.exports = {
 				'unicorn/import-index': 'error',
 				'unicorn/new-for-builtins': 'error',
 				'unicorn/regex-shorthand': 'error',
+				'unicorn/prefer-query-selector': 'error',
```

Comment:
Should be added to the end. You also need to add this to https://github.com/sindresorhus/eslint-plugin-unicorn#usage